packages/slate/src/models/node.js

```diff
@@ -1405,6 +1405,11 @@ class Node {
       return null
     }
 
+    // if we've been given an invalid selection we can exit early.
+    if (!startKey && !endKey) {
```

Comment:
There's actually a `range.isUnset` property for checking this I believe, we should use that instead.